CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# General code owners
+*      @drazik @ptbrowne @doubleface @kosssi
```

Comment:
This comes from `cozy-konnector-template`, I don't know if it must be kept.

Reply:
I don't know who is supposed to maintain this connector in the end